### PR TITLE
Allow Django 5.1

### PR DESCRIPTION
**Pull request by jourdanrodrigues**
Partially addresses https://github.com/soynatan/django-easy-audit/issues/301

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -29,7 +30,7 @@ packages = [{include = "easyaudit"}]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-django = "^4.2"
+django = ">=4.2,<5.1"
```
Comment (mschoettle) [inline pyproject.toml:33]: Did you mean to include 5.1?

```suggestion
django = ">=4.2,<5.2"
```

Comment (jourdanrodrigues) [reply pyproject.toml:33]: YES, omgosh.

**Review by mschoettle — APPROVED** [review]
LGTM (note that this is a community review, I don't have any permissions on this repo)

@samamorgan Is it possible that with the modernize changes the old matrix was removed in CI? There was a build matrix to test the combinations of Python versions with the supported Django versions.

**Comment by jheld** [comment]
sorry i ended up doing this myself, and broadening it to include all of 5.x . version 1.3.7.b3 should be where it's at now.

**Pull request closed without merging**